datafusion/datasource-parquet/src/file_format.rs

```diff
@@ -341,12 +350,28 @@ impl FileFormat for ParquetFormat {
             Some(time_unit) => Some(parse_coerce_int96_string(time_unit.as_str())?),
             None => None,
         };
+        let config_file_decryption_properties = &state
```

Comment:
`infer_stats` below uses the options from `self.options.global` rather than the options for the execution context. Based on the the rest of the code for `ParquetFormat` I think `self.options.global` is the right options to use, and `ParquetFormatFactory` seems to handle setting these to use the config from the execution context if the user hasn't explicitly set options.

Reply:
OK. I think that makes sense. I have changed this.